docs/source/cli_help/github.rst

```diff
@@ -20,6 +20,8 @@ Creates, checks out and manages GitHub PRs while keeping them reflected in branc
   Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
   Equivalent to ``git machete anno --sync-github-prs``.
 
+.. include:: github_config_keys.rst
+
```

Comment:
Pls also replicate the changes in docs.py... in fact I think you can use a shared string there for both `github_config_keys` and `github_api_access` to avoid repetition